test/run_tests.py

```diff
@@ -574,6 +575,12 @@ def filter_csv( values, csv ):
     #[ 'pbcon', gen + dtype + la + n + kd + uplo ],
     ]
 
+# polar
+if (opts.qdwh):
+    cmds += [
+    [ 'qdwh', gen + dtype + n ],
```

Comment:
Can qdwh take rectangular matrices? Then it should be `mn` instead of `n`.

Reply:
Yes, it can be m >= n